h/static/styles/variables.scss

```diff
@@ -42,6 +42,21 @@ $success-color: #1cbd41 !default;
 $mask-start-color: rgba($white, 0) !default;
 $mask-end-color: $white !default;
 
+
+// Variables for the new color palette
```

Comment:
There is another card for restyling the About this version dialog (which uses some of these variables). I'll take off all these variables as part of that card.